docs/standard/serialization/system-text-json-use-dom-utf8jsonreader-utf8jsonwriter.md

```diff
@@ -15,22 +15,18 @@ helpviewer_keywords:
 ms.topic: how-to
 ---
 
-# How to use the DOM, Utf8JsonReader, and Utf8JsonWriter in System.Text.Json
-
-:::zone pivot="dotnet-6-0,dotnet-5-0,dotnet-core-3-1"
-<xref:System.Text.Json.JsonDocument> provides the ability to build a read-only Document Object Model (DOM) by using `Utf8JsonReader`. The DOM provides random access to data in a JSON payload. The JSON elements that compose the payload can be accessed via the <xref:System.Text.Json.JsonElement> type. The `JsonElement` type provides array and object enumerators along with APIs to convert JSON text to common .NET types. `JsonDocument` exposes a <xref:System.Text.Json.JsonDocument.RootElement> property.
-:::zone-end
+# How to use a DOM, Utf8JsonReader, and Utf8JsonWriter in System.Text.Json
 
 :::zone pivot="dotnet-6-0"
-Documentation for mutable DOM support is being developed. Until it's added, see the [.NET 6 Preview 4 announcement](https://devblogs.microsoft.com/dotnet/announcing-net-6-preview-4/#system-text-json-writable-dom-feature).
+> [!IMPORTANT]
+> Some information relates to prerelease product that may be substantially modified before it's released. Microsoft makes no warranties, express or implied, with respect to the information provided here.
 :::zone-end
 
-:::zone pivot="dotnet-6-0,dotnet-5-0,dotnet-core-3-1"
-<xref:System.Text.Json.Utf8JsonReader> is a high-performance, low allocation, forward-only reader for UTF-8 encoded JSON text, read from a `ReadOnlySpan<byte>` or `ReadOnlySequence<byte>`. The `Utf8JsonReader` is a low-level type that can be used to build custom parsers and deserializers. The <xref:System.Text.Json.JsonSerializer.Deserialize%2A?displayProperty=nameWithType> method uses `Utf8JsonReader` under the covers. The  `Utf8JsonReader` can't be used directly from Visual Basic code. For more information, see [Visual Basic support](system-text-json-how-to.md#visual-basic-support).
-
-<xref:System.Text.Json.Utf8JsonWriter> is a high-performance way to write UTF-8 encoded JSON text from common .NET types like `String`, `Int32`, and `DateTime`. The writer is a low-level type that can be used to build custom serializers. The <xref:System.Text.Json.JsonSerializer.Serialize%2A?displayProperty=nameWithType> method uses `Utf8JsonWriter` under the covers.
+This article shows how to use:
 
-The following sections show how to use these APIs for reading and writing JSON.
+* A [JSON Document Object Model (DOM)](#json-dom-choices) for random access to data in a JSON payload.
+* The [`Utf8JsonWriter`](#use-utf8jsonwriter) type for building custom serializers.
+* The [`Utf8JsonReader`](#use-utf8jsonreader) type for building custom parsers and deserializers.
 
 > [!NOTE]
 > The article about migrating from Newtonsoft has more information about these APIs. See the following sections in that article:
```

Comment:
nit: I read this to mean the referenced article has more detailed info about these APIs. Seems weird given that the current one is dedicated to these APIs.